### ISSUE TITLE
Making sidenav backdrop optional?

### ISSUE THREAD

**Issue by IAmJulianAcosta** [miguelcobain/ember-paper#713]
Currently backdrop is shown when sidenav is open (but not locked), could be possible to add an option to make the backdrop optional? 

I will happy to send a PR if this is accepted.

**Comment by v3ss0n** [comment]
I agree , the side nav backdrop slows down sidenav animation in android.
